src/npctalk.cpp

```diff
@@ -1677,18 +1685,17 @@ void dialogue::add_topic( const talk_topic &topic )
 talk_data talk_response::create_option_line( const dialogue &d, const char letter )
 {
     std::string ftext;
-    if( trial != TALK_TRIAL_NONE ) { // dialogue w/ a % chance to work
-        ftext = string_format( pgettext( "talk option", "%1$c: [%2$s %3$d%%] %4$s" ),
-                               letter,                         // option letter
-                               trial.name(),     // trial type
-                               trial.calc_chance( d ), // trial % chance
-                               text                // response
-                             );
-    } else { // regular dialogue
-        ftext = string_format( pgettext( "talk option", "%1$c: %2$s" ),
-                               letter,          // option letter
-                               text // response
-                             );
+    text = truefalse_condition( d ) ? truetext : falsetext;
```

Comment:
Spawned in a pair of NPCs, mind controlled both of them, then tried to talk to one to set it as the overseer, then it crashed on this line.
The way it crashed I can't get the backtrace.

Reply:
Ah, this was something I'd fixed in the final version but didn't make it into the refactoring for submission.

Fixed.